cmd/cli/serve/serve.go

```diff
@@ -250,6 +258,17 @@ func serve(cmd *cobra.Command) error {
 		nodeConfig.RequesterTLSKeyFile = key
 	}
 
+	// Start up Dashboard
+	if startWebUI {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
```

Comment:
If you check the end of this function, you should see 

```
<-ctx.Done() // block until killed
```

which suggests there is already a context you can use.  And at the start of the function you can see it defined.
So, short version, you don't need to create another one here, you can use the one that was already created (just delete these two lines)